src/sbvr-api/permissions.ts

```diff
@@ -387,12 +391,52 @@ const $checkPermissions = (
 	});
 };
 
-const constrainedPermissionError = new PermissionError();
-const generateConstrainedAbstractSql = (
+const convertToLamda = (filter: AnyObject, identifier: string) => {
+	const replaceOject = (object: AnyObject) => {
+		if (_.isString(object)) {
+			return;
+		}
+		if (_.isArray(object)) {
+			object.forEach(element => {
+				replaceOject(element);
+			});
+		}
+
+		if (_.has(object, 'name')) {
```

Comment:
Are all these `_.has` scattered around actually necessary? It feels like unnecessary lodash usage (lodash is great, it's just it's easy to use even when it's not really necessary)